rust/src/write.rs

```diff
@@ -277,6 +293,20 @@ impl<'a, W: Write + Seek> Writer<'a, W> {
             .entry(header.channel_id)
             .or_insert(0) += 1;
 
+        // if the current chunk is larger than our target chunk size, finish it
+        // and start a new one.
```

Comment:
Should we be explicit (both here and in the docs for the option) that we're talking about a target _uncompressed_ size for the chunk? That seems to be the proximal problem in #727 AIUI - readers might want to make sure an uncompressed chunk will fit in a certain amount of memory.